conf/awsbatch.config

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Please remove this - its in nf-core/configs now! :-)

Reply:
Nice!